packages/app/src/app/pages/Pro/legacy-pages/ConfirmBillingInterval.tsx

```diff
@@ -38,79 +44,91 @@ export const ConfirmBillingInterval: React.FC = () => {
-          gap={2}
-          css={css({
-            padding: 4,
-            marginBottom: 8,
-            border: '1px solid',
-            borderColor: 'grays.500',
-            borderRadius: 'small',
-            overflow: 'hidden',
-          })}
-        >
-          <Text size={3}>Team editors</Text>
-          <Stack justify="space-between">
-            <Text variant="muted" size={3}>
-              {seats} {seats === 1 ? 'seat' : 'seats'}
-              <Text size={2}> ✕ </Text>
-              {paymentPreview.nextPayment.currency}{' '}
-              {(paymentPreview.nextPayment.amount / 100 / seats).toFixed(2)}
-            </Text>
-            <Text variant="muted" size={3}>
-              {paymentPreview.nextPayment.currency}{' '}
-              {(paymentPreview.nextPayment.amount / 100).toFixed(2)}
-            </Text>
-          </Stack>
-          <Stack justify="space-between">
-            <Text variant="muted" size={3}>
-              Proration for days left in subscription
-            </Text>
-            <Text variant="muted" size={3}>
-              - {paymentPreview.nextPayment.currency}{' '}
-              {(
-                (paymentPreview.nextPayment.amount -
-                  paymentPreview.immediatePayment.amount) /
-                100
-              ).toFixed(2)}
-            </Text>
-          </Stack>
-          <Stack
-            justify="space-between"
-            css={css({
-              borderTop: '1px solid',
-              borderColor: 'grays.500',
-              paddingTop: 10,
-            })}
+    <div>
+      <Stack gap={10} direction="vertical">
+        <Stack gap={3} direction="vertical" align="center">
+          <Text
+            as="h1"
+            fontFamily="everett"
+            size={48}
+            weight="500"
+            align="center"
+            lineHeight="56px"
+            margin={0}
           >
-            <Text size={3}>Total</Text>
-            <Text weight="bold">
-              {paymentPreview.immediatePayment.currency}{' '}
-              {(paymentPreview.immediatePayment.amount / 100).toFixed(2)} (incl.
-              tax)
-            </Text>
-          </Stack>
+            Change to yearly billing.
+          </Text>
+        </Stack>
+
+        <Stack justify="center">
+          <StyledCard isHighlighted>
+            <Stack direction="vertical" gap={9}>
+              <Stack direction="vertical" gap={6}>
+                <Stack direction="vertical" gap={1}>
+                  <Stack justify="space-between" gap={2}>
+                    <Text>
+                      {subscription.quantity}{' '}
+                      {subscription.quantity === 1 ? 'seat' : 'seats'}
+                    </Text>
+                    {paymentPreview ? (
+                      <Text>
+                        {paymentPreview.nextPayment.currency}{' '}
+                        {(
+                          paymentPreview.nextPayment.amount /
+                          100 /
```

Comment:
what is 100 here? can you move this computation into a variable at the top?

Reply:
Good one. I copied this from the previous implementation but I will add a variable. The `nextPayment.amount` is defined in cents but we want to show it properly.